src/main.cpp

```diff
@@ -11,23 +12,23 @@
 #include <GLFW/glfw3.h>
 
 // Include GLM
-#include <glm/glm.hpp>
-#include <glm/gtc/matrix_transform.hpp>
-#include <glm/gtx/string_cast.hpp>
-
 #include "argh.h"
-
+#include "entity/mesh.hpp"
```

Comment:
What is the include style. If you are going to change it maybe make it consistent.

Reply:
It is consistent

Vendored libs go first